cli/pcluster/utils.py

```diff
@@ -166,11 +166,18 @@ def get_supported_features(region, feature):
     try:
         features = _get_json_from_s3(region, "features/feature_whitelist.json")
         supported_features = features.get("Features").get(feature)
-    except (ValueError, ClientError, KeyError):
-        print(
-            "Failed validate %s. This is probably a bug on our end. Please set sanity_check = false and retry" % feature
-        )
-        exit(1)
+    except (ValueError, ClientError, KeyError) as e:
+        code = e.response.get("Error").get("Code")
+        message = e.response.get("Error").get("Message")
+        if code == "InvalidAccessKeyId":
+            print(message)
+            exit(1)
+        else:
+            print(
+                "Failed validate %s. This is probably a bug on our end. Please set sanity_check = false and retry"
```

Comment:
Shall we be more specific with something like: Failed to retrieve feature whitelisting? Also sanity_check to false doesn't always fix the problem

Reply:
I've changed this too:

```bash
Failed validate efa. This is probably a bug on our end. Please submit an issue https://github.com/aws/aws-parallelcluster/issues/new/choose
```

And yes, you're right, turning off `sanity_check` only works for awsbatch.